src/ODE.jl

```diff
@@ -152,7 +160,7 @@ include("runge_kutta.jl")
 
 # ODE_MS Fixed-step, fixed-order multi-step numerical method
 #   with Adams-Bashforth-Moulton coefficients
-function ode_ms(F, x0, tspan, order::Integer)
+function ode_ms(F, x0, tspan, order::Integer; kwargs...)
```

Comment:
Are these `kwargs...` really necessary?  It just seems like the abstraction is a bit leaky if they now need to accept throw away kwargs.